client/libproject/devstart.rb

```diff
@@ -0,0 +1,17 @@
+require "optparse"
+# How can we tell what symbols come out of this import?
+require_relative "../../libproject/download-swagger-codegen-cli"
+require_relative "../../libproject/utils/common"
+
+def swagger_regen()
+  download_swagger_codegen_cli
```

Comment:
Can this use `Workbench::Swagger.download_swagger_codegen_cli` instead of importing the above?